src/core/utils.js

```diff
@@ -639,18 +630,21 @@ export function getLinkTargets(elem) {
   const linkForElem = elem.closest("[data-link-for]");
   const linkFor = linkForElem ? linkForElem.dataset.linkFor : "";
   const titles = getDfnTitles(elem);
-
-  return titles.reduce((result, title) => {
-    result.push({ for: linkFor, title });
+  const results = titles.reduce((result, title) => {
+    // supports legacy <dfn>Foo.Bar()</dfn> definitions
     const split = title.split(".");
     if (split.length === 2) {
       // If there are multiple '.'s, this won't match an
       // Interface/member pair anyway.
       result.push({ for: split[0], title: split[1] });
     }
-    result.push({ for: "", title });
+    result.push({ for: linkFor, title });
```

Comment:
Why reordering? Is this for some kind of priority thing or just readability?

Reply:
Just readability. `linkFor` is already initialized to the empty string, so might as well use it.